docs/command-reference/server-management/role.md

```diff
@@ -92,4 +92,5 @@ The sentinel output is composed of the following parts:
 ROLE
 ```
 
+// TODO: What about the slave note?
 **A note about the word slave used in this man page**: Starting with Redis 5, if not for backward compatibility, the Redis project no longer uses the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.
```

Comment:
Should we keep such a note that we use the term slave only for backward comp?

Reply:
We should remove this disclaimer.